### PR TITLE
Added My Name To The List

### PR DESCRIPTION
**Pull request by anshulsoni2010**
I have done added my name in the list.

Kindly accept this raised PR and Merge it so that My name will be visible in the portfolio inspiration list.

- [Anshul Soni](https://anshulsoni.in/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +106,6 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 - [Austin Gericke](https://www.austingericke.com)
 - [Austin Pham](https://auspham.dev)
 - [Aycan Öğüt](https://aycan.dev)
-- [Arup Mandal](https://arupmandal.github.io)
```
Comment (pieterjanscheir) [inline README.md:108]: Why was this deleted?

Comment (eclectic-coding) [reply README.md:108]: @pieterjanscheir I am not sure... 

But you should not be adding your request to add your site to another PR

Please remove and open a new request, and I will add

Comment (pieterjanscheir) [reply README.md:108]: I am confused, you just succesfully merged my PR. 

Anyway, I just asked @anshulsoni2010 why he deleted this, assuming it was a mistake.

**Comment by anshulsoni2010** [comment]
Damn!, Maybe I did some mistake there.

**Comment by pieterjanscheir** [comment]
Well, you could make a new PR to correct it I guess @anshulsoni2010

**Comment by anshulsoni2010** [comment]
Thanks for the tip!
I will look.
